docs/src/formulation.md

```diff
@@ -50,19 +50,32 @@ In addition, the following flow sets represent methods for incorporating additio
 | $\mathcal{K}$     | Representative periods (rp)                         | $k \in \mathcal{K}$       | $\mathcal{K} \subset \mathbb{N}$ | $\mathcal{K}$ does not have to be a subset of $\mathcal{P}$                |
 | $\mathcal{B}_{k}$ | Timesteps blocks within a representative period $k$ | $b_{k} \in \mathcal{B}_k$ |                                  | $\mathcal{B}_k$ is a partition of timesteps in a representative period $k$ |
 
+### Sets for Groups
+
+| Name                     | Description             | Elements                       | Superset | Notes |
+| ------------------------ | ----------------------- | ------------------------------ | -------- | ----- |
+| $\mathcal{G}^{\text{a}}$ | Groups of energy assets | $g \in \mathcal{G}^{\text{a}}$ |          |       |
```

Comment:
When I read this I interpret $\mathcal{G}^{\text{a}}$ is a subset of $\mathcal{G}$ filtered by asset $a$, following the notion of  $\mathcal{A}^{\text{i}}$ and $\mathcal{A}$ for example. So I got a bit confused because there, assuming the same notion, we don't have (yet) $\mathcal{G}$ without any superscripts, so to me $\mathcal{G}$ and $\mathcal{G}^{\text{i}}$ is enough. Do we need to differentiate this?

Reply:
Hi @gnawin, Thanks for the comment. I am trying to look here to the big picture, so in the future, we can have groups of flows $G^f$ (It will be helpful also for flow investments but limiting the sum of a group of flows). So, I did it like this to differentiate groups of assets and groups of flows, although groups of flows are not yet in the model. Does that make sense?

Reply:
Yes, it makes sense. I was, only vaguely, thinking of the flows, because I know it would be there but not yet, and you have a better understanding of what you are going to do of course. But my first thought was, for the flows, we still look at the assets, e.g., we only want to sum up all the flows for the group wind. That is why I didn't realize first you would need a separate set. But I can also imagine, that given the Tulipa way, making a set for flows more explicit may be more consistent and clear. 

Just some fruits for thoughts.